test/tests_for_sdkmr/ee2_scheduler_test.py

```diff
@@ -129,85 +129,49 @@ def test_create_submit_file(self):
         #     print(params)
         #     empty_json_sub = c.create_submit(params)
 
-        logging.info("Testing with empty dict as a string ")
-
-        params = self._create_sample_params(cgroups=["{}"])
-
-        empty_json_sub = c._create_submit(params)
-
-        params = self._create_sample_params(cgroups=['{"client_group" : "njs"}'])
-
-        json_sub = c._create_submit(params)
+        logging.info("Testing with regex disabled, bigmem")
 
         params = self._create_sample_params(
-            cgroups=['{"client_group" : "njs", "client_group_regex" : "false"}']
+            {"client_group": "bigmem", "client_group_regex": False}
         )
 
-        json_sub_with_regex_disabled_njs = c._create_submit(params)
-
-        # json_sub_with_regex_disabled
-
-        logging.info("Testing with real valid json ")
-        for sub in [empty_json_sub, json_sub, json_sub_with_regex_disabled_njs]:
-            self.assertEqual(
-                sub[Condor.REQUEST_CPUS], c.config["njs"][Condor.REQUEST_CPUS]
-            )
-            self.assertEqual(
-                sub[Condor.REQUEST_MEMORY], c.config["njs"][Condor.REQUEST_MEMORY]
-            )
-            self.assertEqual(
-                sub[Condor.REQUEST_DISK], c.config["njs"][Condor.REQUEST_DISK]
-            )
-
-        with self.assertRaises(ValueError):
-            logging.info("Testing with real json invalid cgroup {bigmemzlong} ")
-            params = self._create_sample_params(
-                cgroups='{"client_group" : "bigmemzlong", "client_group_regex" : "FaLsE"}'
-            )
-
-            # json_sub_with_regex_disabled
-            c._create_submit(params)
```

Comment:
The job requirements resolver is now responsible for checking invalid client groups